.github/workflows/ok-to-test.yaml

```diff
@@ -0,0 +1,24 @@
+# File based on https://github.com/imjohnbo/ok-to-test/blob/master/.github/workflows/ok-to-test.yml
+name: Ok To Test
+
+on:
+  issue_comment:
+    types: [created]
```

Comment:
Okay I think I might be forgetting but changes to this action wont be run until theyre merged into main correct? If thats the case then my other comment on the PAT doesnt matter

Reply:
Yeah nothing on this workflow will work yet until it's merged into main.